File: 백준/Bronze/25932. Find the Twins/Find the Twins.py
N = int(input())

for _ in range(N):
    nums = list(map(int, input().split(' ')))
    
    print(' '.join(map(str, nums)));
    if 18 in nums and 17 in nums:
        print('both')
    elif 18 in nums:
        print('mack')
    elif 17 in nums:
        print('zack')
    else:
        print('none')
    print()

File: 백준/Bronze/25932. Find the Twins/Find the Twins.js
const input = require('fs').readFileSync(0).toString().trim().split('\n');
const N = Number(input.shift());

for (let i of input) {
    const nums = i.split(' ').map(Number);
    
    console.log(nums.join(' '));
    if (nums.includes(18) && nums.includes(17)) {
        console.log('both');
    } else if (nums.includes(18)) {
        console.log('mack');
    } else if (nums.includes(17)) {
        console.log('zack');
    } else {
        console.log('none');
    }
    console.log();
}